device-types/HPE/Aruba-EC-M-H-AC.yaml

```diff
@@ -1,47 +1,68 @@
 ---
 manufacturer: HPE
-model: Aruba ec-m-h-ac
+model: Aruba EC-M-H AC
 slug: hpe-aruba-ec-m-h-ac
 part_number: '201762'
 u_height: 1
 is_full_depth: true
 airflow: front-to-rear
+weight: 18.1
+weight_unit: lb
+front_image: true
+rear_image: true
 console-ports:
   - name: console
-    type: de-9
+    type: rj-45
     label: console
 power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.